Casks/v/vscodium.rb

```diff
@@ -1,20 +1,30 @@
 cask "vscodium" do
   arch arm: "arm64", intel: "x64"
 
-  version "1.98.1.25070"
-  sha256 arm:   "6463f2beb0a834e59da6d386e797a09f6b33e0d089b93995df09257a1ef58377",
-         intel: "dcaa9e1a73cc5ae93c4cb82971e0b457b168f42f1e07c05df806bf7ce9252039"
+  on_catalina :or_older do
+    version "1.97.2.25045"
+    sha256 arm:   "48d01a0663b7a6396f41ddc11296eb812d58e9fe3b671b9d33e6b21621e40f21",
+           intel: "af8fe5721ef431ab59fe05a06a3a462c40884229f61cc2962ea01d3e66997243"
+    livecheck do
```

Comment:
Extra line for readability - and aligns with the `on_big_sur` block in style

```suggestion
           intel: "af8fe5721ef431ab59fe05a06a3a462c40884229f61cc2962ea01d3e66997243"

    livecheck do
```

Reply:
Is this a known bug that `brew style` doesn't fix?

I ran the following command before submitting this PR:
```bash
brew style --fix Casks/v/vscodium.rb
1 file inspected, no offenses detected
```